public/API/API_GetGameExtended.php

```diff
@@ -8,6 +8,7 @@
  *  string     Title                      name of the game
  *  int        ConsoleID                  unique identifier of the console associated to the game
  *  string     ConsoleName                name of the console associated to the game
+ *  string     ParentGameID               unique identifier of the parent game if this is a subset
```

Comment:
shouldn't this be `int?`

Reply:
Yes, it should be. Fixed in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/c384be10eec370633b6d37d8f2c33714547b872f.